Sources/SwiftGtk/enums/PackDirection.swift

```diff
@@ -0,0 +1,48 @@
+import CGtk
+
+/// Determines how widgets should be packed inside menubars and menuitems contained in menubars.
+/// 
+/// https://docs.gtk.org/gtk3/enum.PackDirection.html
+public enum PackDirection {
+    /// Widgets are packed left-to-right.
+    case ltr
```

Comment:
I think it would be a good idea to rename these to `leftToRight` (and similarly for the other cases) for better readability.